crawl-ref/source/initfile.cc

```diff
@@ -250,6 +250,7 @@ const vector<GameOption*> game_options::build_options_list()
         new BoolGameOption(SIMPLE_NAME(show_travel_trail), USING_DGL),
         new BoolGameOption(SIMPLE_NAME(use_fake_cursor), USING_UNIX ),
         new BoolGameOption(SIMPLE_NAME(use_fake_player_cursor), true),
+        new BoolGameOption(SIMPLE_NAME(show_rampage_indicator), true),
```

Comment:
I'm going to cut this until someone actually asks for it. Better fewer options than more :)